subworkflows/local/arg.nf

```diff
@@ -21,6 +24,21 @@ workflow ARG {
      // Prepare HAMRONIZATION reporting channel
     ch_input_to_hamronization_summarize = Channel.empty()
 
+    // AMRfinderplus run
+    if ( !params.arg_skip_amrfinderplus ) {
+
+        AMRFINDERPLUS_UPDATE ( )
```

Comment:
We need to make this optional, in some cases a user may need to run the pipeline on offline nodes, so have to be able to manually supply the database field. Will also need download description instructions in `usage.md` too.

Reply:
Ok, I'll start working on the option